internal/db/seeds/dao_test.sql

```diff
@@ -4,7 +4,7 @@
 BEGIN;
 
 INSERT INTO accounts(id, account_number, org_id)
-VALUES (1, '13', '000013')
+VALUES (1, '13', '1')
```

Comment:
Although WithTenant does not fail on this today, it should in the future.
We have DefaultOrgId = "1" and default_account_number = "1", now we are doing it wrong and looking just by orgId, but hopefully we will improve on it and then test would start to fail :)

```suggestion
VALUES (1, '1', '1')
```